osu.Game/Screens/Ranking/ScorePanel.cs

```diff
@@ -206,7 +207,12 @@ private void updateState()
             switch (state)
             {
                 case PanelState.Expanded:
-                    Size = new Vector2(EXPANDED_WIDTH, expanded_height);
+                    var height = expanded_height;
```

Comment:
Same goes for this. Your PR should not touch anything outside of your mod classes. Anything else that needs changing should be its own PR which has zero dependencies on your mod.

Reply:
Alright, but before I submit another PR: Should the seed even be displayed at the result screen? I don't want to implement something that makes it possible for mods to modify the result screen when it ends up never being used, because this mod is probably the only use case for something like this.

![](https://user-images.githubusercontent.com/47356629/118182585-7a203a00-b439-11eb-9a2a-7f2655f8777e.png)

Reply:
Mod icons have tooltips that are supposed to be showing the mod customisation options set for a given score.

Leaderboards for this sort of mod are pretty much useless in the current state of things anyway. And making them not useless for random is a whole lot of work, outside the scope of this PR.

Reply:
I didn't know about the tooltips to be honest. That makes this redundant of course.